safe-global/web-core
# fix: warn if `safeTxGas` >= block gas limit

## What it solves

Resolves #1795

## How this PR fixes it
Places a warning in the create transaction modal if `safeTxGas` is greater than or equal to the last block gas limit

## How to test it
1. Create a staking Thales transaction on Optimism and observe the warning in the modal.
2. The Simulation should fail with error code GS010

## Screenshots
![Screenshot 2023-05-03 at 21 04 56](https://user-images.githubusercontent.com/32431609/236018212-d83714fb-9755-48cf-ad0d-e9067ea91025.png)

## Checklist
